### PR TITLE
Added impl of DataTime<Utc> to Apiv2Schema

### PR DESCRIPTION
**Pull request by sfisol**

### CODE REVIEW

File: core/src/v2/actix.rs [hunk 1]
```diff
@@ -155,6 +155,8 @@ mod manual_impl {
     impl_simple!(usize);
     #[cfg(feature = "datetime")]
     impl_simple!(chrono::NaiveDateTime);
+    #[cfg(feature = "datetime")]
+    impl_simple!(chrono::DateTime<chrono::Utc>);
```
Comment (wafflespeanut) [inline core/src/v2/actix.rs:159]: Can we implement this for generic `DateTime<T>` rather than just for `DateTime<Utc>`?

**Comment by wafflespeanut** [comment]
Closed and reopened so that Travis picks it up.

**Comment by sfisol** [comment]
Done, but keep in mind that Swagger will render it wrongly as UTC (f.ex. "2020-05-26T10:14:40.337Z") just as it renders it wrongly for NaiveDateTime :-)

**Comment by wafflespeanut** [comment]
Note that the rendering doesn't have anything to do with the plugin itself (it's the serde impl). We only implement a trait for emitting the schema.

**Pull request merged**